perf/CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 set(CMAKE_CXX_STANDARD 14)
 
+set(PERF_OUTPUT_DIR ${PROJECT_BINARY_DIR}/perf_output)
+file(MAKE_DIRECTORY ${PERF_OUTPUT_DIR})
```

Comment:
How about `perf/output` instead of `perf_output` on the top level?

Reply:
Replaced.